build-pre-commit.xml

```diff
@@ -137,7 +148,7 @@
     </target>
 
     <target name="php-license" depends="get-changeset.php-spacesep" if="changeset.php.notempty">
-        <exec executable="/bin/grep" failonerror="true" outputproperty="php-license.grep.output">
+        <exec executable="/bin/grep" failonerror="false" outputproperty="php-license.grep.output">
```

Comment:
It still fail, sir. When it is set to true, the build just fails on that step. When false, it shows a list of files without licence headers.